clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go

```diff
@@ -205,11 +165,15 @@ func (w *waitForControlledPodsRunningMeasurement) start() error {
 	w.isRunning = true
 	w.stopCh = make(chan struct{})
 
-	w.informer, err = w.newInformer()
-	if err != nil {
-		klog.Infof("%v: create dynamic informer failed.", w)
-		return err
-	}
+	w.informer = informer.NewDynamicInformer(
+		w.clusterFramework.GetDynamicClients().GetClient(),
+		w.gvr,
+		w.namespace,
+		w.fieldSelector,
+		w.labelSelector,
+		w.queue,
```

Comment:
Why do we need to pass w.queue? I would assume that it's enough to pass handleObject function (and that already has access to everything it needs).

Reply:
To go back to that - I think that w.queue shouldn't be part of this interface.
If you need to add to the queue as part of handler, handler function should be doing that.
But I can imagine certain cases where we don't want to do that and we shouldn't force users to it.

Reply:
Queue removed.